docs/layers.md

```diff
@@ -1395,6 +1400,11 @@ Railway `service` values are:
 
 **Piers** and **quays** start showing up at zoom 13+ with `kind_detail` values of `pier` and `quay`, respectively. If mooring information is available, the `mooring` property will be one of `no`, `yes`, `commercial`, `cruise`, `customers`, `declaration`, `ferry`, `guest`, `private`, `public`, `waiting`, `yacht` or `yachts`.
 
+#### Roads hgv `kind_detail` values
+
+For `hgv`the `kind_detail` values are `no`, `designated`, `destination`, `delivery`, `local` and `agricultural`.
+For `hgv_restriction` the `kind_detail` values are `weight`, `height`, `length`, `width`, `wpa` (weight per axle), `kpra` (king pin to rear axle leght, in cm.), `hazmat`, `other` and `multiple` if more than one.
```

Comment:
Since people always ask, let's specify units as (kg) and (m) next to each value, please...

Reply:
Will post an update with it